examples/langgraph-memanto/README.md

```
@@ -0,0 +1,49 @@
+
+# 2. Set your API key
+cp .env.example .env
+# Edit .env with your MOORCHEH_API_KEY
+
+# 3. Run the simulation
+python agent.py
+```
+
+## Example Output
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)